ext/fs/lib.rs

```diff
@@ -51,3 +51,66 @@ pub async fn op_remove_dir(#[string] path: String, recursive: bool) -> Result<()
 
     Ok(())
 }
+
+#[op2(async)]
+pub async fn op_copy_file(#[string] origin: String, #[string] dest: String) -> Result<(), AnyError> {
+    tokio::fs::copy(origin, dest).await?;
+    Ok(())
+}
+
+#[op2(async)]
+pub async fn op_copy_dir(#[string] origin: String, #[string] dest: String) -> Result<(), AnyError> {
+    op_copy_dir_recurse(origin, dest).await?;
+    Ok(())
+}
+
+use async_recursion::async_recursion;
+#[async_recursion]
+pub async fn op_copy_dir_recurse(origin: String, dest: String) -> Result<(), AnyError> {
```

Comment:
this is not really an `op`, prefix functions with `op_` only when they use `#[op]`/`#[op2]` macros